infra/ansible/roles/django/templates/settings.yaml.j2

```diff
@@ -54,3 +54,19 @@ YOUTUBE_API_KEY: {{youtube_api_key}}
 
 ENABLE_API_WIKIDATA:
     MIGRATIONS: {{django_enable_api_wikidata_migrations}}
+
+TWITTERBOT_CREDENTIALS:
+    "@TournesolBotFR": {
+        "LANGUAGE": "fr",
+        "CONSUMER_KEY": "{{consumer_key_twitterbot_fr}}",
+        "CONSUMER_SECRET": "{{consumer_secret_twitterbot_fr}}",
+        "ACCESS_TOKEN": "{{access_token_twitterbot_fr}}",
+        "ACCESS_TOKEN_SECRET": "{{access_token_secret_twitterbot_fr}}",
+    }
+    "@TournesolBot": {
+        "LANGUAGE": "en",
+        "CONSUMER_KEY": "{{consumer_key_twitterbot_en}}",
+        "CONSUMER_SECRET": "{{consumer_secret_twitterbot_en}}",
+        "ACCESS_TOKEN": "{{access_token_twitterbot_en}}",
+        "ACCESS_TOKEN_SECRET": "{{access_token_secret_twitterbot_en}}",
+    }
```

Comment:
I think that Ansible will fail when using this template including new variables that are not defined in the inventory yet. 

I would recommend to move this change to #702, to be able to merge and deploy these 2 PRs step by step.